docs/get-started/authenticate-without-flask-strava.md

```diff
@@ -0,0 +1,414 @@
+```
+
+
+```python
+from stravalib.client important Client
+# You will use webbrowser to launch a browser from Python
+import webbrowser
+```
+
+Below, you read in the `client_id` and `access_token` from the 
+`client_secrets.txt` file that you created above. You then create a 
+stravalib `Client()` object.
+
+The `Client()` object is what you will use to get and push data to Strava.
+It stores your authentication information inside of it which allows data transfer 
+from (and to) to happen based upon the scope of the token that you request. 
+
+We will discuss token scope below. 
+
+
+```python
+# Open the secrets file and store the client_id and secret as objects
+# Note that .read() reads the file, .strip() gets rid of extra spaces and 
+# split, splits the file (2) elements at the comma into two Python objects.
+client_id, access_token = (
+        open("client_secrets.txt").read().strip().split(",")
+    )
+# Next, generate a stravalib Client() object
+client = Client()
+```
+
+## Authenticate with Strava
+
+Now you have all of the pieces needed to actually authenticate with Strava. 
+
+1. You have the client secret and token from your Strava account's developer app and   
+2. You have the stravalib `Client()` object.
+
+
+### Create a url to authenticate and define the token scope 
+
+Next you will need to create two variables:
+
+1. The scope of the authentication token that you wish to receive from Strava: 
+Essentially the scope refers to permissions that you want to have when interacting 
+with your users Strava data. For example if you only wish to download data, then read level permissions
+are enough. However if you wish to push data to Strava (for example upload activities) then you will also 
+need write access. 
+
+2. You will need to specify a URL (`redirect_url`) that strava will redirect the user to after they 
+have authenticated (signed in and provided the permissions that you requested in 
+step one. If you are only authenticating locally to grab your own data, you can 
+use a localhost value for your `redirect_url`. 
+
+```{tip}
+However, if you are building an app, then your redirect url might be the 
+url of your app. You will need to grab the code that Strava provides in that 
+`redirect_url`. 
+```
+
+### Set your authentication scope & create a redirect url
+
+If you only wish to download your data, then your scope can be read only (see 
+below). The scope below provides your code with access to the users 
+profile data and activity data. Notice that all of the scope values are `read`:
+
+```python
+# Python list containing read only scope values
+scope = ["read_all", "profile:read_all", "activity:read_all"]
+```
+However, if you wish to upload data to Strava then you will need a write scope 
+included in your URL. Like this: 
+
+```python
+# Python list containing both read and write (activity) scope values
+request_scope = ["read_all", "profile:read_all", "activity:write", "activity:read_all"]
+```
+
```

Comment:
Maybe provide a link here to the strava documentation on all available scopes?

E.g., https://developers.strava.com/docs/authentication/#details-about-requesting-access

Reply:
excellent idea! will add!

Reply:
added!

Reply:
done